parsec/backend/webhooks.py

```diff
@@ -3,7 +3,8 @@
 import trio
 from typing import Optional
 from structlog import get_logger
-from urllib.request import urlopen, Request, URLError
+from urllib.error import URLError
```

Comment:
Is there a reason for this ?

Reply:
That's [where it lives](https://docs.python.org/3.8/library/urllib.error.html#urllib.error.URLError).

Also, mypy.